src/types.rs

```diff
@@ -456,3 +456,12 @@ impl Default for ChannelConfig {
 		LdkChannelConfig::default().into()
 	}
 }
+
+/// Custom TLV entry.
```

Comment:
Should it be named CustomTlvEntry instead of adding as a comment?

Reply:
I felt we could use a more elaborate comment, like elsewhere in the library, but I also feel I like expertise to document it properly 😅

Reply:
No problem. Let's see what feedback we get from LDK.